include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -322,7 +322,9 @@ struct SessionOptions : Base<OrtSessionOptions> {
 
   SessionOptions& AddConfigEntry(const char* config_key, const char* config_value);
   SessionOptions& AddInitializer(const char* name, const OrtValue* ort_val);
-  OrtStatus* OrtSessionOptionsAppendExecutionProvider_CUDA(OrtSessionOptions* options, OrtCUDAProviderOptions* cuda_options);
+
+  SessionOptions& AppendExecutionProvider_CUDA(const OrtCUDAProviderOptions& provider_options);
+  SessionOptions& AppendExecutionProvider_OpenVINO(const OrtOpenVINOProviderOptions& provider_options);
```

Comment:
Please, document all the new APIs. You can use standard XML that IDE can generate or any other style Doxygen can consume.

Reply:
Gotcha, I followed the pattern now.